packages/reactor-api/src/processors/operational-processor.ts

```diff
@@ -0,0 +1,22 @@
+import { IAnalyticsStore } from "@powerhousedao/analytics-engine-core";
+import { Document, OperationScope } from "document-model/document";
+import { Processor } from "./processor";
+import { ProcessorOptions, ProcessorSetupArgs } from "src/types";
+import { Knex } from "knex";
```

Comment:
Use the Db type from "utils/get-db-client" instead. Maybe we should rename that file to `utils/db`